.circleci/config.yml

```
@@ -1397,7 +1397,8 @@ jobs:
       - matrix_notify_failure_unless_pr
 
   t_ems_ext_edr:
-    <<: *base_node_small
+    # Runs OOM on the small instance
```

At first I thought you forgot an `into` here. Don't run OOMs on our instances :P

```suggestion
    # Runs out of memory on the small instance
```

---

Fine, but only because it's you!